azure-pipelines.yml

```diff
@@ -111,7 +111,16 @@ stages:
 
           - task: PublishBuildArtifacts@1
             displayName: 'Publish Artifact: drop'
-
+   
+      - job: Terraform
+        pool:
+          vmImage: ${{ variables.agentPool }}
+        steps:
+        - task: CmdLine@2
+          displayName: Setup Terraform
+          inputs:
+            script: 'curl https://github.com/hmcts/cnp-azuredevops-libraries/scripts/tfenv-install-terraform.sh'
```

Comment:
you need to run the script in each job so that terraform gets loaded.

you can use this to load the library:
https://github.com/hmcts/sds-jenkins-infrastructure/blob/master/azure-pipelines.yml#L7-L13

(make sure you use the master branch not the PTL hack